src/database/database-dummy.cpp

```diff
@@ -80,3 +80,19 @@ void Database_Dummy::listPlayers(std::vector<std::string> &res)
 		res.emplace_back(player);
 	}
 }
+
+bool Database_Dummy::getPairs(const std::string &modname, StringMap *storage)
```

Comment:
Shouldn't all of these work in-memory (currently they are no-op)?

Reply:
Yes. Right now these no-ops work fine because of the way `ModMetadata` interacts with the database (it keeps its own data in-memory.)